pytorch_to_returnn/torch/nn/modules/activation.py

```diff
@@ -40,6 +40,17 @@ class LogSigmoid(_ActivationReturnn):
   func_name = "log_sigmoid"
 
 
+class Power(Module):
```

Comment:
This is not in Torch, right?
Then you need to add this:
```
is_original_torch_module = False
```

Reply:
Yes, right.